src/RTMP.as

```diff
@@ -171,54 +171,59 @@ package {
     }
 
     private function playerPlay(url:String=null):void {
-      if (!mediaElement) {
-        if (isLive) {
-          urlResource = new StreamingURLResource(url, StreamType.LIVE);
-        } else {
-          urlResource = new StreamingURLResource(url, StreamType.RECORDED);
-        }
+      try {
```

Comment:
I think this is not wrong, but we could generalize a little bit the errors to let someone knows where it happens.